lib/pusher.js

```diff
@@ -109,17 +102,16 @@ const async = require('async'),
         });
     };
 
-    function copyFilesToDst(session, src, dst, dstType, options, next) {
+    function copyFilesToDst(session, src, dst, dstType, options, middleCb, next) {
         log.verbose("pusher#copyFilesToDir()","src:", src, ", dst:", dst);
-        const self = this;
 
         const successMsg = function(_src, _dst, _size, _ignore) {
             if (!_ignore) {
                 spinner.stop();
-                console.log("Push:", _src, "->", _dst);
+                middleCb("Push:" + _src + " -> " + _dst);
```

Comment:
Do you recommend writing "+ _src" as ${src}, right?

This change is a stylistic change because this part is different from the rest of the code.
I understand what you recommended, but it's hard to change everything.